connector.rb

```diff
@@ -529,17 +535,17 @@
           nil,
           input['query']
         )
-        documents = parsed_query&.[](:documents)
-        if documents.present?
-          operation = documents.find { |doc| doc[:operation_name] == input['operation_name'] }
-        end
+        docs = parsed_query&.[](:documents)
```

Comment:
Is there a guarantee that `documents` object is either an array or nil (not present). That it can never be a scalar?

Reply:
Yes, it is.